index.js

```diff
@@ -195,9 +195,13 @@ function RingPop(options) {
     this.serverRate = new metrics.Meter();
     this.totalRate = new metrics.Meter();
 
-    // 10.30.8.26:20600 -> 10_30_8_26_20600
-    this.statHostPort = this.hostPort.replace(/\.|:/g, '_');
-    this.statPrefix = 'ringpop.' + this.statHostPort;
+    if (options.statPrefix) {
```

Comment:
Maybe explicitly check for `undefined`. So `false` can be used to NOT emit a stat prefix